wls-admin-service/src/test/java/de/muenchen/oss/wahllokalsystem/adminservice/MicroserviceApplicationIntegrationTest.java

```diff
@@ -0,0 +1,47 @@
+            //WireMock-Stubbing is served from json files in mapping directory
+
+            val headers = new HttpHeaders();
+            headers.add("Authorization", "Bearer " + tokenToSend);
+            val request = new RequestEntity<>(headers, HttpMethod.POST, new URI("http://localhost:" + port + "/endpoint/does/not/matter"));
+
+            val result = restTemplate.exchange(request, Void.class);
+
+            Assertions.assertThat(result.getStatusCode()).isEqualTo(HttpStatus.UNAUTHORIZED); //unauthorized cause token is expired
```

Comment:
wo in dem test wird sichergestellt, dass die max-header-size gesetzt ist? 
> Wenn die Property nicht gesetzt kommt es zu einem BadRequest und nicht zu unauthorized

müsste man nicht einen test hinzufügen der die Assertion `...isEqualTo(HttpStatus.BAD_REQUEST);` hat?

Reply:
wenn der Header nicht gesetzt kommt es zu dem Bad-Request. Ich hatte überlegt ob ich den negativ Test schreiben. Allerdings müsste dann der Header in einer seperaten yml-Datei gesenkt werden. Dadurch dass die Settings in `application.yml` vorliegen greift der `max-http-request-header-size` immer, solange es nicht überschrieben wird. Ich habe mich daher dafür entschieden zu prüfen ob unsere Headgröße ausreichend ist. Daher keinen negativen Test.